.github/CONTRIBUTING.md

```diff
@@ -418,26 +418,26 @@ for(var/atom/thing as anything in bag_of_atoms)
 
-* Queries must never specify the database, be it in code, or in text files in the repo.
+* Queries should never specify the database, be it in code, or in text files in the repo.
 
-* Primary keys are inherently immutable and you must never do anything to change the primary key of a row or entity. This includes preserving auto increment numbers of rows when copying data to a table in a conversion script. No amount of bitching about gaps in ids or out of order ids will save you from this policy.
+* Primary keys are inherently immutable and you should never do anything to change the primary key of a row or entity. This includes preserving auto increment numbers of rows when copying data to a table in a conversion script. No amount of bitching about gaps in ids or out of order ids will save you from this policy.
 
-* The ttl for data from the database is 10 seconds. You must have a compelling reason to store and reuse data for longer then this.
+* The ttl for data from the database is 10 seconds. You should have a compelling reason to store and reuse data for longer then this.
 
 * Do not write stored and transformed data to the database, instead, apply the transformation to the data in the database directly.
 	* ie: SELECTing a number from the database, doubling it, then updating the database with the doubled number. If the data in the database changed between step 1 and 3, you'll get an incorrect result. Instead, directly double it in the update query. `UPDATE table SET num = num*2` instead of `UPDATE table SET num = [num]`.
 	* if the transformation is user provided (such as allowing a user to edit a string), you should confirm the value being updated did not change in the database in the intervening time before writing the new user provided data by checking the old value with the current value in the database, and if it has changed, allow the user to decide what to do next.
 
 ### Mapping Standards
 * Adding, Removing, or Replacing Station Maps
-	* All pull requests adding, removing, or replacing station maps must receive prior approval from a maptainer, or they will be closed without additional warning.
+	* All pull requests adding, removing, or replacing station maps should receive prior approval from a maptainer, or they will be closed without additional warning.
 
 * TGM Format & Map Merge
-	* All new maps submitted to the repo through a pull request must be in TGM format (unless there is a valid reason present to have it in the default BYOND format). This is done using the [Map Merge](https://tgstation13.org/wiki/Map_Merger) utility included in the repo to convert the file to TGM format.
+	* All new maps submitted to the repo through a pull request should be in TGM format (unless there is a valid reason present to have it in the default BYOND format). This is done using the [Map Merge](https://tgstation13.org/wiki/Map_Merger) utility included in the repo to convert the file to TGM format.
```

Comment:
TGM should always be used, it is better at handling conflicts and functions exactly the same in the eyes of BYOND. There is no reason to ever not use this